spec/dynamodb_framework/dynamodb_table_manager_spec.rb

```diff
@@ -117,6 +117,29 @@
 
   end
 
+  it 'can update the TTL attribute' do
+
+    exists = subject.exists?('update_ttl_test')
+
+    if exists
+      subject.drop('update_ttl_test')
+    end
+
+
+    builder = DynamoDbFramework::AttributesBuilder.new
+    builder.add(:id, :S)
+    builder.add(:name, :S)
+    builder.add(:number, :N)
+    builder.add(:ttl_date, :N)
+
+    subject.create('update_ttl_test', attributes_builder.attributes, :id)
```

Comment:
I'm not sure what this test is testing?
 - `builder` is being setup up lines 129..133 but then not used
 - No checks that the ttl attribute has been updated from false to true that I can see.